project/src/test/java/com/google/sps/data/PlacesTest.java

```diff
@@ -123,6 +106,56 @@ public void filter_filterNotRequired_notFiltering() {
     assertEquals(result, allPlaces);
   }
 
+  @Test
+  public void filter_filterByWebsitePresence() {
+    ImmutableList<Place> placesToKeep = ImmutableList.of(
+        // Place has both a website URL and a google URL
+        createValidPlaceBuilderByName("name1").build(),
+        // Place has a website URL and doesn't have a google URL
+        createValidPlaceBuilderByName("name2").setGoogleUrl("").build(),
+        // Place doesn't have a website URL and has a google URL
+        createValidPlaceBuilderByName("name3").setWebsiteUrl("").build()
+    );
+    // Place doesn't have a website URL or a google URL
+    Place placeToFilter =
+        createValidPlaceBuilderByName("name4").setWebsiteUrl("").setGoogleUrl("").build();
+    ImmutableList<Place> allPlaces =
+        ImmutableList.<Place>builder().addAll(placesToKeep).add(placeToFilter).build();
+
+    ImmutableList<Place> result = Places.filter(
+        allPlaces /* places */,
+        1 /* min rating */,
+        true /* filter if no website */,
+        false /* filter branches of same place */
+    );
+
+    assertEquals(result, placesToKeep);
+  }
+
+  @Test
+  public void filter_filterByStatus() {
+    Place placeToKeep = createValidPlaceBuilderByName("name1")
+        .setBusinessStatus(BusinessStatus.OPERATIONAL).build();
+    ImmutableList<Place> placesToFilter = ImmutableList.of(
```

Comment:
Why two places? One is enough to test the functionality IMO.

Reply:
Sorry, I'm not sure I understand your question :) 
Each place has a different status, and I wanted to make sure that we keep only the places that have the desired statuses

Reply:
Sorry, for some reason I thought two of them have the same status...